client/src/component/CohortForm.js

```diff
@@ -0,0 +1,82 @@
+import { useState } from "react";
+// import { Link } from "react-router-dom";
+import { Button } from "./Button";
+import { InputeField } from "./InputeField";
+
+export const CohortForm = () => {
+    const [formInput, setFormInput] = useState({
+        name: "",
+        start_date: "",
+        m_1: "",
+        m_2: "",
+        m_3: "",
+        m_4: "",
+        m_5: "",
+        m_6: "",
+        m_7: "",
+        m_8: "",
+    });
+
+    const AddCohort = async (input) => {
+        console.log(input);
+        try {
+            const response = await fetch("http://localhost:3000/api/cohorts", {
```

Comment:
This will work locally and on the server
```suggestion
            const response = await fetch("/api/cohorts", {
```

Reply:
thank you done